fetch_tags.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+###
+# EXIT CODES
+# 0 - success
+# 1 - parameter error
+# 50 - dockerhub pull error
+# 51 - no valid docker image tag
+# 52 - no old tag file found
+# 53 - max tag count iteration reached
+# 100 - new tag found, pr_script triggered
+###
+
+# Sample envionment variables
+# IMAGE_ID="weaver-worker"
+# DOCKERHUB_REPO="pavics/weaver"
+# TAG_FILTER="^[0-9]+(\\.[0-9]+)*$"
+# BUMP_TAG="WEAVER_WORKER_IMAGE"
+# BUMP_FILE="birdhouse/default.env"
+
+REQUIRED_ENV_VARS='
+    IMAGE_ID
+    DOCKERHUB_REPO
+    TAG_FILTER
+    BUMP_TAG
+    BUMP_FILE
+'
+
+# args parsing
+for env_var in $REQUIRED_ENV_VARS
+do
+    if [[ ! -v "${env_var}" ]]; then
+        echo "[ERROR] [$0] Missing ${env_var} environment variable. Exiting."
+        exit 1
+    fi
+done
+
+echo "[STEP] [$0] [$IMAGE_ID] Fetch tags"
+
+DATA_DIR="data"
```

Comment:
Harcode to "data"?  It's not re-using `HISTORIC_TAG_DATA_PATH`?

Reply:
`DATA_DIR` doesn't change, it's always in `data/` that we store the data (latest tag fetched, for each image).
However, `HISTORIC_TAG_DATA_PATH` is the path on which we want to mount a docker volume, which will point to `data/`.